tket2-hseries/src/extension/hseries.rs

```diff
@@ -393,20 +448,21 @@ mod test {
         let hugr = {
             let mut func_builder = FunctionBuilder::new(
                 "all_ops",
-                Signature::new(vec![QB_T, FLOAT64_TYPE], vec![QB_T, BOOL_T]),
+                Signature::new(vec![QB_T, FLOAT64_TYPE], vec![BOOL_T]),
             )
             .unwrap();
             let [q0, angle] = func_builder.input_wires_arr();
-            let q1 = func_builder.add_qalloc().unwrap();
+            let q1 = func_builder.build_qalloc().unwrap();
```

Comment:
It would be nice if these methods were consistently named to begin with `add_`, rather than some beginning with `build_`, unless there's a significant difference?

Reply:
`add` is for primitives (following other `add` methods in builder traits) and `build` is for building _in terms_ of primitives